podcast/elixir-wizards/index.html

```diff
@@ -73,7 +85,32 @@ <h2>Season Ten: The Next Ten Years of Elixir</h2>
 
     <section class="section content-block podcast" id="s9">
         <h2>Season Nine: Parsing the Particulars</h2>
-        <p>Our theme in Season Nine is Parsing the Particulars. We'll be digging into the details about a specific topic of interest to our guests.</p>
+
+        <p>
+          The theme for Elixir Wizards Season 9 is "Parsing the Particulars." Hosts Sundi,
+          Owen, Dan, and Bilal take a deep dive into the intricacies of Elixir
+          development. The Wizards explore critical aspects of the Elixir programming
+          language, including: observability, GenServers, geomapping, Phoenix LiveView,
+          Oban, Tailwind, and Nerves.
+        </p>
+
+        <iframe
+          src="https://player.fireside.fm/v2/IAs5ixts+kKtocYfA?theme=dark"
+          width="740"
+          height="200"
+          frameborder="0"
+          scrolling="no"
+        ></iframe>
+
+        <blockquote cite="/podcast/elixir-wizards/s9-e11-chris-miller-programming-language/">
+          Inherently, language is about communicating ideas and communicating ideas is
+          like abstraction essentially, right? When you think about what a programming
+          language is, it's a general abstraction over computation
+        </blockquote>
+
+        <cite>
```

Comment:
this cite appears too close to the next paragraph

![image](https://github.com/smartlogic/smartlogic.io/assets/122999/c1bd9a0e-ef68-4233-ac50-863686731fe9)